src/typeDefs/queries.ts

```diff
@@ -39,6 +39,10 @@ export const queries = gql`
       skip: Int
     ): [Donation!]!
 
+    getFundCampaigns: [FundCampaign]
```

Comment:
No need for this. These should be implemented as a field level resolver either on the `Organization` model, or the other relevant parent models to `FundCampaign`